source/developers-guide/plugin-configuration/index.md

```diff
@@ -173,6 +173,45 @@ Below all supported configuration elements including their design and source cod
 
 <img src="img/numberfield.png" alt="Numberfield" class="image-border" />
 
+### Boolean
+To define something similar like checkbox you can use `boolean`
+```xml
+<elements>
+    <element required="false" type="boolean" scope="shop">
```

Comment:
just state only the required things here. label, name, etc are not special for the boolean type

Reply:
Of course they may not be required but for more transparency when viewing the code and not start trying to guess what could be used, I really would recommend to keep these lines. I could add a sentence to point out that they are writen for transparency and not because of requirement.